src/Module.php

```diff
@@ -318,26 +333,46 @@ public function renderToolbar($event)
 
     /**
      * Checks if current user is allowed to access the module
+     * @param \yii\base\Action|null $action
      * @return bool if access is granted
      */
-    protected function checkAccess()
+    protected function checkAccess($action = null)
```

Comment:
How can it be null?

Reply:
E.g. when it's called from here: https://github.com/yiisoft/yii2-debug/blob/4cc02c14f0e204e5895a124df59da07d0b3a48dc/src/Module.php#L321

Reply:
or here:
https://github.com/yiisoft/yii2-debug/blob/4cc02c14f0e204e5895a124df59da07d0b3a48dc/src/Module.php#L279